Runtime/Reporters/SlackMessageSender.cs

```diff
@@ -107,16 +107,20 @@ public SlackMessageSender(SlackAPI api)
                     slackToken,
                     slackChannel,
                     withoutAlpha.EncodeToPNG(),
-                    postTitleTask.Ts
+                    postTitleTask.Ts,
+                    cancellationToken
                 );
                 if (!postScreenshotTask.Success)
                 {
                     return;
                 }
             }
 
-            var body = Body(logString, stackTrace);
-            await _slackAPI.Post(slackToken, slackChannel, body, postTitleTask.Ts);
+            if (stackTrace != null && stackTrace.Length > 0)
```

Comment:
@nowsprinting Sorry for my late reply. I’m wondering if it’s better to send it as proposed or to use the default message. Could you please explain why you chose not to use the default message?

Reply:
This post is 2nd or 3rd post.
1st post: `logString`
2nd post: screenshot if flag is true

Before this PR, 3rd post body is equivalent to 1st post if `stachTrace` is empty.

Reply:
Ah, I understood.